latest/examples/multiscales_strict/.config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "schema": "schemas/strict_image.schema"
```

Comment:
This almost makes me wonder whether this configuration file could be structured like the JSON schema test suite with some relative pointers to the real data e.g.

```json
{
    "description": "TBD",
    "schema": "schemas/strict_image.schema",
    "tests": [
        {
            "description": "TBD",
            "data": {
              "$ref": "multiscales_example.json"
            }
        },
                {
            "description": "TBD",
            "data": {
              "$ref": "multiscales_transformations.json"
            }
        }
    ]
}
```
 
 The primary downside is that we will have to remember to actively register new examples. The current implementation automatically detects them via `glob`

Reply:
Fair enough. :+1: for lowering the burden of extraction